ast/spec.md

```diff
@@ -791,6 +791,7 @@ enum BinaryOperator {
      | "|" | "^" | "&" | "in"
      | "instanceof"
      | "|>"
+     | "??"
```

Comment:
This is a `LogicalOperator`, not a `BinaryOperator`

Reply:
https://github.com/babel/babylon/pull/762